compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -3371,6 +3371,244 @@ static void arrayCopyDefault(TR::Node* node, uint8_t elementSize, TR::Register*
       }
    }
 
+/*
+ * Generate a store instruction
+ * Parameters:
+ * node: the tree node
+ * addressReg: the base register for the destination mem address
+ * index: the index for the destination mem address
+ * valueReg: the value that needs be stored
+ * size: the size of valueReg
+ * cg: the code generator
+ */
+static void generateStoreInstructions(TR::Node* node, TR::Register* addressReg, int32_t index, TR::Register* valueReg, uint8_t size,  TR::CodeGenerator* cg)
```

Comment:
Awesome that you wrote documentation for newly added functions! It would be great if you could Doxygen this function comment so as to stay consistent with other parts of our codebase. There are numerous examples around if you grep. Similarly elsewhere in this commit.